lib/cb/utils/validator.rb

```diff
@@ -64,6 +63,16 @@ def try_parse_xml(body)
       rescue MultiXml::ParseError
         nil
       end
+
+      def error_message(processed_response)
+        if processed_response['errors']
+          processed_response['errors']
+        elsif processed_response['Errors']
+          processed_response['Errors']
+        else
+          ''
+        end
```

Comment:
Sorry about not seeing this comment before. The definition of `#fetch` is: `fetch(key [, default])`

So we can get a little fancy with it and use the default to do the alternative lookup or just return empty string:

`processed_response.fetch('errors', processed_response('Errors', ''))`